src/main/java/com/dataloom/datastore/util/RefreshTokenAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,68 @@
+package com.dataloom.datastore.util;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+
+import com.auth0.spring.security.api.Auth0AuthenticationEntryPoint;
+import com.auth0.spring.security.api.Auth0TokenException;
+import com.dataloom.exceptions.ErrorsDTO;
+import com.dataloom.exceptions.LoomExceptions;
+import com.dataloom.organizations.roles.exceptions.TokenRefreshException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Returns json error messages. The format of error messages here has to be consistent with
+ * {@link DataStoreExceptionHandler}. Modified from {@link Auth0AuthenticationEntryPoint}
+ * 
+ * @author Ho Chung Siu
+ *
+ */
+public class RefreshTokenAuthenticationEntryPoint implements AuthenticationEntryPoint {
+    private ObjectMapper mapper;
+
+    public RefreshTokenAuthenticationEntryPoint( ObjectMapper mapper ) {
+        this.mapper = mapper;
+    }
+
+    @Override
+    public void commence(
+            HttpServletRequest request,
+            HttpServletResponse response,
+            AuthenticationException authException ) throws IOException, ServletException {
+        response.setContentType( "application/json" );
```

Comment:
Just a suggestion... perhaps use one of the Spring constants instead of the `"application/json"` string literal?